### DISCUSSION TITLE
[Question][Hw4] About updating seeds

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#565] [Q&A]
In the lecture, it is said to store mutations that increases the code coverage. But for the given json_parser, I noticed that seed_2.json works perfectly well, and covers all of the code but seed_1.json is buggy and covers less then seed_2.json. So every time I give a mutation of seed_2.json that has no bug the seed corpus is updated, and I think this is not what the assignment intended. Am I getting something wrong?

**Comment by RiceBiscuits** [comment]
What does 'Seed_1.json is buggy' mean?

**Comment by ee12ha0220** [comment]
Never mind, I was just foolish....
